tests/test_throttle.py

```diff
@@ -281,6 +281,24 @@ def test_changing_settings(self):
         assert allowed is True
         assert tokens == 99
 
+    @pytest.mark.parametrize('value', ['a', -100, '-100'])
```

Comment:
Also, now that I'm nitpicking, it might be worth adding some positive test cases here too. I don't see any other explicit `throttle_set` test cases in this file

I've done something like this in the past:
```python
@pytest.mark.parametrize(
	'value,raises',
	[('a', True), (-100, True), ('-100', True), ('0', False), ...],
)
def test_x(self, value, raises):
	# setup ...
	if raises:
		with pytest.raises(...):
	else:
		# doesn't raise
		# ...
```

Reply:
`test_changing_settings` uses successful throttle sets to test. So, I'll merge this for now, but we can always revisit it later if that isn't enough.